src/analysis/Environments/Environments.test.ts

```diff
@@ -78,8 +70,9 @@ const listRuntimesV2: () => Promise<ListRuntimeItem[]> = jest.fn();
 const listWithoutProject: () => Promise<ListAppResponse[]> = jest.fn();
 const list: () => Promise<ListDiskItem[]> = jest.fn();
 const mockFetchLeo = jest.fn();
-const defaultNav = {
-  getLink: jest.fn().mockReturnValue(''),
+const defaultNav: NavLinkProvider<EnvironmentNavActions> = {
+  getUrl: jest.fn().mockReturnValue(''),
```

Comment:
This doesn't seem like a valid default implementation. I might have this throw instead to make that clear.

Reply:
just to clarify, this is just a mock for the unit tests.  I can rename defaultNav to mockNav, and have it return '/' to look like a valid route.

Reply:
fixed to return '/'